src/components/flow/routers/webhook/WebhookRouterForm.tsx

```diff
@@ -79,14 +79,14 @@ export default class WebhookRouterForm extends React.Component<
     let ensureEmptyHeader = false;
 
     if (keys.hasOwnProperty('method')) {
+      const oldMethod = this.state.method.value.value;
+      const newMethod = keys.method.value;
       updates.method = { value: keys.method };
 
-      if (keys.method.value !== GET_METHOD.value) {
-        if (!this.state.postBody.value) {
-          updates.postBody = { value: DEFAULT_BODY };
-        }
-      } else {
-        updates.postBody = { value: null };
+      if (oldMethod === Methods.GET && newMethod !== Methods.GET) {
+        updates.body = { value: DEFAULT_BODY };
+      } else if (oldMethod !== Methods.GET && newMethod === Methods.GET) {
+        updates.body = { value: null };
```

Comment:
This should probably be calling `getDefaultBody(method)` since that returns an empty string which will not match later equivalence checks.

Reply:
updated to always use `getDefaultBody`